microsoft-graph-spring-social-starter/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>azure-spring-boot-starter-parent</artifactId>
+        <groupId>com.microsoft.azure</groupId>
+        <version>0.1.4</version>
+        <relativePath>../azure-spring-boot-starter-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>microsoft-graph-spring-social-starter</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Microsoft Graph Spring Social Starter</name>
```

Comment:
I think it's better to have a readme under starter folder to explain what's this starter for, and basic functionality. although professional user could guess some point from naming. it's still necessary to have such readme.